rust/src/client.rs

```diff
@@ -235,6 +235,23 @@ pub(crate) fn client_render_download_progress(
     }
 }
 
+pub(crate) fn microdnf_install(arg: &str) -> Result<()> {
+    let mut packages;
+    if !arg.starts_with("install"){
```

Comment:
See comment below; I think what we want here is to take a an array of (borrowed) strings i.e. `args: &[&str]` like e.g. `cliwrap.rs` does with `pub fn entrypoint(args: &[&str]) -> Result<()> {`.